imagersite/imager_images/views.py

```diff
@@ -1,3 +1,43 @@
-from django.shortcuts import render
+"""Views for imager_images."""
 
-# Create your views here.
+from django.shortcuts import render, redirect, get_object_or_404
+from .models import Album, Photo
+from django.contrib.auth.models import User
+
+
+def library_view(request, username=None):
```

Comment:
The library view should not have the `username` as a parameter in the url, thus you will not need this keyword argument.